src/FormInput.tsx

```diff
@@ -62,15 +62,19 @@ const FormInputValidation: React.FunctionComponent<IFormInputValidationProps> =
 
     setValidations(
       compact(
-        validationDescription.tests.map(t => {
+        validationDescription.tests.map((t: any) => {
           const params: any = t.params || {};
           if (validationDescription.type === 'string') {
             if (t.name === 'min') {
               return `at least ${params.min} letters`;
             } else if (t.name === 'max') {
               return `at most ${params.max} letters`;
             }
+
+            return null;
           }
+
+          return null;
```

Comment:
I added these nulls because tslint was failing the build since not all code paths returned a value.  Since we're using `_.compact` on the map result then those nulls will be removed from the final result anyway.